partcad-cli/src/partcad_cli/click/commands/supply/find.py

```
@@ -0,0 +1,68 @@
+import rich_click as click
+from partcad import logging
+
+import asyncio
+from partcad.provider_data_cart import ProviderCart
+import partcad as pc
+import json
+from typing import Any, List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)